feat/src/main/java/com/example/feat/document/components/TimeRichText.kt

```diff
@@ -0,0 +1,56 @@
+package com.example.feat.document.components
+
+import androidx.compose.foundation.layout.Box
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.res.colorResource
+import androidx.compose.ui.text.AnnotatedString
+import androidx.compose.ui.text.SpanStyle
+import androidx.compose.ui.text.font.FontWeight
+import androidx.compose.ui.text.style.TextAlign
+import androidx.compose.ui.unit.sp
+import com.example.feat.R
+
+@Composable
+fun TimeRichText(
+    header: String,
+    hours: Long,
+    minutes: Long,
+    seconds: Long? = null,
+) {
+    Box(
+        contentAlignment = Alignment.Center // Center the content within the Box
+    ) {
+        val contents = buildList<String> {
+            add(header)
+            add(" $hours ")
+            add("hours, ")
+            add("$minutes ")
+            add("minutes ")
+            if (seconds != null) {
+                add("and ")
+                add("$seconds ")
+                add("seconds")
+            }
```

Comment:
The `buildList` function creates a new list instance every time, which may not be necessary if the list size is known beforehand. Consider using `mutableListOf` for better performance.

```diff
-        val contents = buildList<String> {
+        val contents = mutableListOf<String>().apply {
```

Reply:
I think you have a misunderstanding. 
- `buildList` : is applied for immutable list => Optimized for list creation
- `mutableListOf`: is applied for mutable list => Standard list initialization